Include/README.rst

```diff
@@ -0,0 +1,63 @@
+The Python C API
+================
+
+The C API is divided into three sections:
+
+1. ``Include/``
+2. ``Include/cpython/``
+3. ``Include/internal/``
+
+
+Include: Limited API and Stable ABI
+===================================
+
+``Include/`` contains the public Limited API and the Stable ABI.  The
+Stable ABI is defined by :pep:`384`.
+
+Functions stealing references or returning borrowed references *must not*
+be added to the Limited API API or the Stable ABI.  A strong reference
+*must* be returned.  The API in ``Include/`` should not expose or leak
+implementation details.
```

Comment:
I'd prefix this with a marker like **NOTE:** and maybe indent the whole paragraph. Or perhaps insert a section subheading before it explaining that here are some rules for making additions or changes to the limited API.

PS: "Limited API API".